src/usernotificationsui.cs

```diff
@@ -83,6 +83,23 @@ interface NSExtensionContext_UNNotificationContentExtension {
 
 		[Export ("mediaPlayingPaused")]
 		void MediaPlayingPaused ();
+
+		[NoWatch, NoTV, NoMac, iOS (12,0)]
+		[Export ("performNotificationDefaultAction")]
+		void PerformNotificationDefaultAction ();
+
+		[NoWatch, NoTV, NoMac, iOS (12,0)]
+		[Export ("dismissNotificationContentExtension")]
+		void DismissNotificationContentExtension ();
+
+		// property, but we have to add the two methods
```

Comment:
Maybe add "because this is a category" here.

Reply:
@rolfbjarne valid point, we never know who is going to read the code. Will push a small change with that.